### ISSUE TITLE
[MAS4.3.1][High Contrast Mode-Restart Conversation] After applying high contrast mode, focus inside the "Restart Conversation" dropdown is not visible.

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1912]
**Actual Result:**
When applying High Contrast Settings of windows or application, we observe that the focus visibility in the "restart conversation" dropdown is very low.

**Expected Result:**
When we apply any High Contrast Settings of windows or application, the focus visibility should be consistent as well as the user will be able to identify the presence of focus inside the "restart conversation" dropdown.

**Repro Steps:**
Turn on "Invert Color" setting from accessibility settings or follow below mentioned path to turn on Application High Contrast setting:
File->Themes->High Contrast
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the "Open Bot" button and activate it.
4. Open a Bot Dialog box gets opened.
5. Navigate to the "bot URL" edit box and enter URL- http://localhost:3978/api/messages
6. Navigate to "Connect" button and activate it.
7. "Live Chat" separate window appears.
8. Navigate to "Menu Bar" control.
9. Expand "Restart Conversation" dropdown.
9. Observe the issue.

**Note:**
Same issue observed in Linux, when we turn on the high contrast mode feature of the application. As well as in normal mode the focus visibility is also very low. Please refer attachment: "Focus visibility in High Contrast Mode_Linux"
Turn on High Contrast Mode from: File-->Themes-->High-contrast

**User Impact:**
Users with low vision use high contrast settings to access content present in an application.

**Test Environment:**
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-80829)

OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.5.2-linux

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74}](url)

**Attachment:**
[Focus visibility in High Contrast Mode.zip](https://github.com/microsoft/BotFramework-Emulator/files/3695786/Focus.visibility.in.High.Contrast.Mode.zip)
[Focus visibility in High Contrast Mode_Linux.zip](https://github.com/microsoft/BotFramework-Emulator/files/3798011/Focus.visibility.in.High.Contrast.Mode_Linux.zip)

**Comment by ShikhaMishra11** [comment]
We will validate and update here.

**Comment by ShikhaMishra11** [comment]
Verified the issue and it is fixed in latest build.
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (version: 4.6.0-92058)

OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.6.0-92058-linux-x86_64
